tests/baselines/reference/isolated-declarations/auto-fixed/diff/declarationEmitDefaultExportWithStaticAssignment.d.ts.diff

```diff
@@ -0,0 +1,99 @@
+// [[Reason: Function declarations are not fixed]] ////

```

Comment:
Generally speaking, can we rephase these messages from "are not fixed" to "are not supported by isolatedDeclarations" or similar? "not fixed" seems pretty unclear to me as to whether or not it's a bug or just not a feature we support yet.